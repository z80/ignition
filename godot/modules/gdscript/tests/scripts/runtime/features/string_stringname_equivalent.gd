# https://github.com/godotengine/godot/issues/64171

func test():
	print("Compare ==: ", "abc" == &"abc")
	print("Compare ==: ", &"abc" == "abc")
	print("Compare !=: ", "abc" != &"abc")
	print("Compare !=: ", &"abc" != "abc")

	print("Concat: ", "abc" + &"def")
	print("Concat: ", &"abc" + "def")
	print("Concat: ", &"abc" + &"def")
