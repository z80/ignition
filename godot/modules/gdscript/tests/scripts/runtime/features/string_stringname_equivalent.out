GDTEST_OK
Compare ==: true
Compare ==: true
Compare !=: false
Compare !=: false
Concat: abcdef
Concat: abcdef
Concat: abcdef
